CMakeLists.txt

```diff
@@ -29,6 +29,9 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(openthread-br VERSION 0.3.0)
 
+add_compile_options(-Wall -Wextra -Werror -Wfatal-errors -Wuninitialized -Wno-missing-braces)
+set(CMAKE_CXX_FLAGS_DEBUG_INIT "")
+set(CMAKE_CXX_FLAGS_RELEASE_INIT "-O2")
```

Comment:
The default flag seems to be `-O3 -DNDEBUG`

We are removing -DNDEBUG ?

Reply:
Good point. I've removed these lines. Now the debug build goes with `-g` while the release build goes with `-O3 -DNDEBUG`.